### PR TITLE
[DOC-7039] remove bulk ops mention

### PR DESCRIPTION
**Pull request by ashleykwtam**
Addresses DOC-7039:
- remove bulk ops mention in the file storage section for DR

Preview:
- [Use Userfile Storage](https://deploy-preview-16350--cockroachdb-docs.netlify.app/docs/dev/use-userfile-storage.html)
- [Use Local File Server](https://deploy-preview-16350--cockroachdb-docs.netlify.app/docs/stable/use-a-local-file-server.html)

### CODE REVIEW

**Review by nickvigilante — APPROVED** [review]
LGTM!

**Comment by kathancox** [comment]
Hey Ashley, I think this looks great, thanks for doing so quickly! Looking at the naming, I wonder if "Use Userfile Storage" 
 is easier to read and more descriptive for the page? And maybe: "Use a Local File Server for Storage" (Though I think "Use a Local File Server" stands fine on it's own. Let me know what you think.

**Comment by ashleykwtam** [comment]
Sounds good @kathancox! I've changed it to "Use Userfile Storage" so the title is more descriptive and can better explain the page content. I've kept "Use a Local File Server" as I think this works on its own

File: v22.2/use-userfile-storage.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
 ---
-title: Use Userfile for Bulk Operations
+title: Use Userfile Storage
 summary: The IMPORT statement imports various types of data into CockroachDB with user-scoped storage.
```
Comment (kathancox) [inline v22.2/use-userfile-storage.md:3]: Ah, looks like the summary here is incorrect... not sure when this got added. Strange! Could you update? Maybe something like the following, adapt as you see fit.

> Store data from CockroachDB operations in your cluster's userfile storage.

File: v22.2/use-a-local-file-server.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
 ---
-title: Use a Local File Server for Bulk Operations
-summary: Learn how to create a simple file server for use with bulk operations within CockroachDB
+title: Use a Local File Server
+summary: Learn how to create a simple file server for use with operations within CockroachDB
```
Comment (kathancox) [inline v22.2/use-a-local-file-server.md:3]: The end of this summary is a little awkward, do you mind changing it while you're working here? Perhaps, something like the following:

> Learn how to create a simple file server to use with CockroachDB operations.

File: v23.1/use-userfile-storage.md [hunk 3]
```diff
@@ -1,11 +1,12 @@
 ---
-title: Use Userfile for Bulk Operations
+title: Use Userfile Storage
 summary: The IMPORT statement imports various types of data into CockroachDB with user-scoped storage.
```
Comment (kathancox) [inline v23.1/use-userfile-storage.md:3]: Same here!

File: v23.1/use-a-local-file-server.md [hunk 4]
```diff
@@ -1,7 +1,8 @@
 ---
-title: Use a Local File Server for Bulk Operations
-summary: Learn how to create a simple file server for use with bulk operations within CockroachDB
+title: Use a Local File Server
+summary: Learn how to create a simple file server for use with operations within CockroachDB
```
Comment (kathancox) [inline v23.1/use-a-local-file-server.md:3]: Same here!

**Review by kathancox — APPROVED** [review]
LGTM, with a couple of small nits on the two summaries in the frontmatter of the pages. Thanks!

**Pull request merged**